src/main/java/org/hl7/davinci/priorauth/ProcessClaimItemTask.java

```diff
@@ -142,4 +173,23 @@ private boolean process() {
 
         return ret;
     }
+
+    private Disposition sendAndGetDisposition(Bundle bundle, int seq, String address) throws IOException {
```

Comment:
Something I just noticed here is `seq` is not used. The internal rules engine finds the disposition of each individual item. But it looks like here the external rules engine is not made aware of the which item we are requesting the disposition for. I see two options:
1. Run the external disposition once and then parse the result for each item
2. Change this call to pass the sequence in (proposing something like `{address}?seq={seq}`) and make sure the external disposition engine is set up to accept that request. 

Since this is out of scope of the IG we get to design the interface between PAS and the rules engine. Thus I would lean towards the second approach unless there is good reason to do the first.

Reply:
I agree we should take the 2nd approach